src/doc/reference.md

```diff
@@ -555,17 +555,33 @@ Users of `rustc` can define new syntax extensions in two ways:
 
 ## Macros
 
-`macro_rules` allows users to define syntax extension in a declarative way.  We
-call such extensions "macros by example" or simply "macros" — to be distinguished
-from the "procedural macros" defined in [compiler plugins][plugin].
+`macro_rules` allows users to define syntax extensions in a declarative way.
+We call such extensions "`macro_rules!` macros" or simply "macros" — to be
+distinguished from the "procedural macros" defined in [compiler plugins][plugin].
```

Comment:
The term ‶macros by example″ has also been used at several places and might be worth emntionning.